crates/egui/src/widgets/text_edit/builder.rs

```diff
@@ -201,6 +203,13 @@ impl<'t> TextEdit<'t> {
         self
     }
 
+    /// Set the background color of the [`TextEdit`].
```

Comment:
Maybe you can add a comment that by default this comes from `Visuals::extreme_bg_color`?